src/documentation/pages/components/spinner/examples/_sizes-lumen-centurylink.vue

```diff
@@ -0,0 +1,108 @@
+<template lang="pug">
+  <ComponentExample title="Sizes" titleSize="h4" id="sizes-lumen-centurylink" :tabs="exampleTabs">
+    .chi-grid.-align-items--end(slot="example")
+      div.-p--2(v-for="size in ['xs', 'sm', 'sm--2', 'sm--3', 'md', 'lg', 'xl', 'xxl']")
+        chi-spinner(:size="size", class='-m--1')
+        p.-text.-text--center {{size}}
+    <Wrapper slot="code-webcomponent">
+      .p--text.chi-tab__description
```

Comment:
p.-text

Reply:
Actually p.-text is not required. .chi-tab_description takes care of it.